addons/account/data/digest_data.xml

```diff
@@ -5,4 +5,19 @@
             <field name="kpi_account_total_revenue">True</field>
         </record>
     </data>
+
+    <data>
```

Comment:
Why using a separate data tag ? Could be in the same noupdate="1" section as the digest record.